MudTable: Horizontal scrolling can now be enabled with HorizontalScrollbar parameter

boukenka opened the pull request:
Implementation of a _HorizontalScrollbar_ parameter for [#1174](https://github.com/Garderoben/MudBlazor/issues/1174).
There is a demonstration of the use of the parameter on the _TableHorizontalScrollbarTest page_ in the _MudBlazor.UnitTests.Viewer_ project.

boukenka committed:
HorizontalScrollbar parameter addition for MudTable

boukenka committed:
Typo: 'an horizontal' instead of 'a horizontal'

boukenka committed:
Fix possible style collision

henon commented:
@felinepc please take a look if that solves your problem.

henon commented:
@mkalinski93 please note that your original PR is being changed to opt-in via parameter by this PR. Please check if you care

boukenka committed:
Update of 'Table with horizontal Scrolling' example in documentation

boukenka commented:
I deleted the TableHorizontalScrollbarTest page in the MudBlazor.UnitTests.Viewer. I think it's not really necessary.
I updated the 'Table with horizontal scrolling' example in the official documentation. It was necessary because it was not working anymore.

henon commented on src/MudBlazor.Docs/Pages/Components/Table/TablePage.razor:
```diff
@@ -154,7 +154,7 @@
                 <Title>Table with horizontal Scrolling</Title>
                 <Description>
-                    If too many columns are displayed, the table will allow the user to scroll the content horizontally
+                    If too many columns are displayed, setting <CodeInline>HorizontalScrollbar="true"</CodeInline> allow the user to scroll the content horizontally
```
Here the word `will` got dropped

boukenka replied:
Modification done

henon reviewed (changes requested):
Just a minor wording fix required, then this can be merged.

boukenka committed:
Fix wording

boukenka commented:
@henon Thank you!
